api/src/test/java/org/pmiops/workbench/reporting/ReportingMapperTest.java

```diff
@@ -26,16 +30,33 @@
   public static class conifg {}
```

Comment:
?

Reply:
renamed. These classes are basically never used by name (since annotation finds them), but `Configuration` is our standard name.